src/components/community/button-community.js

```diff
@@ -0,0 +1,40 @@
+import '@brightspace-ui/core/components/link/link.js';
+import '@brightspace-ui/core/components/button/button-subtle.js';
+import { html, LitElement } from 'lit';
+import { communityUrlFactory } from './directive/community-article.js';
+import { LanguageListenerController } from '../../controllers/language-listener/language-listener.js';
+
+class CommunityLink extends LitElement {
+
+	static get properties() {
+		return {
+			text: { type: String },
+			articleMap: { attribute: 'article-map', type: Object }
+		};
+	}
```

Comment:
If you want, you could create a base component that has the properties, constructor with `langController` setup and `connectedCallback` and then both would extend that and just need to provide `render()`.